Formula/disque.rb

```diff
@@ -0,0 +1,17 @@
+class Disque < Formula
+  desc "Disque, an in-memory, distributed job queue"
+  homepage "https://github.com/antirez/disque"
+  url "https://github.com/antirez/disque/archive/1.0-rc1.tar.gz"
+  version "1.0-rc1"
```

Comment:
Yeah, seems so. I’ll consider the devel tap. Thanks!

From:  Mike McQuaid <notifications@github.com>
Reply-To:  Homebrew/homebrew-core <reply@reply.github.com>
Date:  Thursday, April 21, 2016 at 1:05 PM
To:  Homebrew/homebrew-core <homebrew-core@noreply.github.com>
Cc:  Jacek Szpot <maligree@gmail.com>, Author <author@noreply.github.com>
Subject:  Re: [Homebrew/homebrew-core] disque 1.0-rc1 (new formula) (#481)


In Formula/disque.rb:
> @@ -0,0 +1,17 @@
> +class Disque < Formula
> +  desc "Disque, an in-memory, distributed job queue"
> +  homepage "https://github.com/antirez/disque"
> +  url "https://github.com/antirez/disque/archive/1.0-rc1.tar.gz"
> +  version "1.0-rc1"
We'll need to pass on this for now, then, sorry. Perhaps a candidate for https://github.com/homebrew/homebrew-devel-only?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub